articles/active-directory/cloud-infrastructure-entitlement-management/onboard-gcp.md

```diff
@@ -65,37 +65,39 @@ There are several moving parts across GCP and Azure, which are required to be co
     Optionally, specify **G-Suite IDP Secret Name** and **G-Suite IDP User Email** to enable G-Suite integration.
 
 
-1. You can either download and run the script at this point or you can do it in the Google Cloud Shell.
+1. You can either download and run the script at this point or you can run it in the Google Cloud Shell.
 
 1. Select **Next** after successfully running the setup script. 
 
 Choose from three options to manage GCP projects. 
 
 #### Option 1: Automatically manage 
 
-The automatically manage option allows projects to be automatically detected and monitored without extra configuration. Steps to detect list of projects and onboard for collection:  
+The automatically manage option allows projects to be automatically detected and monitored without extra configuration. Requirements to detect list of projects and onboard for collection:  
 
-1. Firstly, grant **Viewer** and **Security Reviewer** role to service account created in previous step at organization, folder or project scope. 
+1. In GCP, you will need to grant **Viewer** and **Security Reviewer** role to service account created in previous step at a project, folder or organization. 
 
-To enable controller mode 'On' for any projects, add following roles to the specific projects:
-- Role Administrators
-- Security Admin 
+ > [!NOTE]
+ > To enable controller mode 'On' for any projects, add following roles to the specific projects:
+ > - Role Administrators
+ > - Security Admin 
```

Comment:
To enable Controller mode **On** for any project, add these roles to the specific projects:
- Role Administrators
- Security Admin